### DISCUSSION TITLE
Multiple Lines for Tags

### DISCUSSION THREAD

**Discussion by Irodzuita** [gotson/komga#1807] [Ideas]
Please make it so that the tags line in the comic/one-shot details page can have more than one scrollable line. It makes it quite hard to get a good idea of the comic based on the tags when I can only see like 2 or 3 at a time on the phone or when theres a lot it can just be iritating to scroll through. 

Ideally for me there would be a settings option for a N max rows after which overflow continues to become scrollable as it does currently just with the N number rows and tags spread between them. I understand that you probably don't want a ton of clutter on this screen but I think it would be quite helpful for quickly looking through new comics that have been uploaded.

**Comment by gotson** [comment]
Duplicate of #1585

**Comment by Irodzuita** [comment]
That is now 4 years old. This is to once again say that there is a need for a change in this area. Not exactly satisfied with the last response of "With a long list it would take the whole screen, not very convenient." and explain another possible solution / reasoning behind why it would be benifitial. Just brushing it off as a duplicate of something unchange from a post so old is a bit irritating. I like this app but some customization for QOL would be nice
